Libs/vtkSegmentationCore/vtkBinaryLabelmapToClosedSurfaceConversionRule.h

```diff
@@ -56,6 +56,10 @@ class vtkSegmentationCore_EXPORT vtkBinaryLabelmapToClosedSurfaceConversionRule
   /// Joint smoothing converts all segments in shared labelmap together, reducing smoothing artifacts.
   static const std::string GetJointSmoothingParameterName() { return "Joint smoothing"; };
 
+  // Conversion methods
+  static const std::string CONVERSION_METHOD_FLYING_EDGES;
+  static const std::string CONVERSION_METHOD_SURFACE_NETS;
```

Comment:
Ideally this should be available from Python but given current limitation of the VTK wrapper, it is not ... instead of adding methods for this, I will look into improving VTK.

That said, I would prefer to do this later and tackle this in a follow-up PR.